spec/controllers/trains_controller_spec.rb

```diff
@@ -28,7 +28,7 @@
   # Train. As you add validations to Train, be sure to
   # adjust the attributes here as well.
   let(:valid_attributes) do
-    { number: '12345', current_station: create(:railway_station) }
+    { number: '12345', current_station_id: create(:railway_station).id, routes: [create(:route)] }
```

Comment:
Please use one style creating relations this or that https://github.com/OmegaLord/studybook-RoR/pull/4/files#diff-a795ac331a96ea9e96779fb1815c98b2R99